tests/lib/nm/ipv4_test.py

```diff
@@ -48,7 +49,8 @@ def test_create_setting_without_addresses(NM_mock):
         config={
             'enabled': True,
             'address': [],
-        }
+        },
+        base_con_profile=None
```

Comment:
What is this?

Reply:
It's `nm.ipv4.create_setting(config={ 'enabled': True, 'address': [],}, base_con_profile=None)`